doc/INSTALL.html

```diff
@@ -1011,12 +1012,12 @@
 </PRE></div>
 
 <P>
-I suggest you also add C:\Qt\4.8.0\bin\ to your Environment Variables Path in
+I suggest you also add C:\Qt\5.9.1\bin\ to your Environment Variables Path in
 the windows system preferences.
 </P>
 <P>
 If you plan to do some debugging, you'll need to compile debug version of Qt:
-C:\Qt\4.8.0\bin\qtvars.bat compile_debug
+C:\Qt\5.9.1\bin\qtvars.bat compile_debug
 </P>
 <P>
 Note: there is a problem when compiling debug version of Qt 4.7, the script ends with
```

Comment:
I guess we no longer need information for Qt 4.7 ;)

Reply:
I pondered that line... but in the end I'm not sure what to do with it. I don't know if it's still an issue with recent Qt, and I don't want to just blindly change it to Qt 5 if it's no longer required. I figured leaving it at 4.7 would slightly hint to builders that maybe it's not needed, but if something goes wrong they could try it.

Like I noted, I didn't verify any of this (nor do I plan to go through the pain of a mingw build. Building with msvc is horrible enough).

Reply:
fair enough.

are you going to convert the t2t files to something usable like md or rst? :P

Reply:
No.

Well... maybe I'll trade. If you translate expression help from json, i'll think about this. ;)

Reply:
It [looks](https://github.com/qgis/QGIS/commit/6c006671516dab876e974ad8a337c3e6e4528774) like [this is still true](https://github.com/qgis/QGIS/blob/096b80e44462760728b7cb93d86132d312ebe2ed/INSTALL#L83)?

```
/!\ Note to document writers: This document is generated from 
doc/INSTALL.t2t - if you need to edit this document, be sure to edit that 
file rather than the generated INSTALL document found in the root of the 
source directory.
```

What version of txt2tags are you using?
https://github.com/txt2tags/txt2tags

(I apologize if the workflow is posted somewhere and I missed it.)

Reply:
I could use a hint how to regenerate INSTALL from msvc.t2t over in #36212

Reply:
I found my inspiration [here](https://github.com/qgis/QGIS/blob/61f1566285a36073a883eb4681dc5d05d1e1b5f2/scripts/release.pl#L185). 🙂